grid-compose/pkg/types/types.go

```diff
@@ -0,0 +1,70 @@
+package types
+
```

Comment:
All of these fields require validation e.g valid flist url, positive disk space, valid node id, depends_on a service and so on.